tests/test_radio_dish.cpp

```diff
@@ -395,15 +395,14 @@ static bool is_multicast_available (int ipv6_)
 
     msleep (SETTLE_TIME);
 
-    rc = sendto (send_sock, msg, static_cast<socklen_t> (strlen (msg)), 0,
```

Comment:
strlen() returns size_t and socklen_t is int -> size reduction.

Reply:
`man sendto` says:

       #include <sys/socket.h>

       ssize_t send(int sockfd, const void buf[.len], size_t len, int flags);
       ssize_t sendto(int sockfd, const void buf[.len], size_t len, int flags,
                      const struct sockaddr *dest_addr, socklen_t addrlen);
       ssize_t sendmsg(int sockfd, const struct msghdr *msg, int flags);

The third parameter to sendto is size_t. The third parameter to recvfrom (line 405) has no cast.

Reply:
I have realized that the function is different on windows. And that the `len` parameter is an `int` there. But it is not a `socklen_t`. I have updated the PR to use tha same kind of `#ifdef` as is used in other parts of teh ZMQ code.